src/v1/controllers/RegistrationController.js

```diff
@@ -94,6 +94,7 @@ export default BaseLoginController.extend({
   registerUser: function(postData) {
     const self = this;
 
+    Object.keys(postData).forEach((k) => postData[k] === null && delete postData[k]);
```

Comment:
Should we assert for `undefined` too?
`Object.keys(postData).forEach((k) => (_.isNull(postData[k]) || _.isUndefined(postData[k]) && delete postData[k]);`

Please add a test for this.

Reply:
This is fixing a bug, and it means we do not have a test for this specific scenario, otherwise we should have catched this before. Should we add an e2e test too?

Please add a video/screenshot for the fix.

One more thing, since this is fixing a bug, I think you will need to base your branch on the latest release branch I think it is `6.5` . One more thing, you will need to run downstream test on monolith.

Reply:
I agree with you, e2e will be too much 👍

Reply:
+1 to changing your target branch to `6.4` since this is a bug fix! Also agree with checking for undefined and adding a unit test

Reply:
@jonathanhansen-okta I have one more question, is that possible that the `postData[k]` could be empty string? is this something we need to validate too? I am just curios

Reply:
Addressed all your comments
- added check for undefined and empty
- added unit tests
- added video for fix
- added monolith bacon test results
- changed target branch to 6.4

Reply:
Thank you so much @jonathanhansen-okta, great work!!